src/index.ts

```diff
@@ -0,0 +1,130 @@
+import * as path from "path"
+import * as fs from "@ts-common/fs"
+import * as md from "@ts-common/commonmark-to-markdown"
+import * as azureMd from "@azure/openapi-markdown"
+import * as ai from "@ts-common/async-iterator"
+import * as jp from "@ts-common/json-parser"
+import * as it from "@ts-common/iterator"
+import * as json from "@ts-common/json"
+import * as sm from "@ts-common/string-map"
+
+export const cli = async <T>(f: (path: string) => AsyncIterable<T>): Promise<number> => {
+  try {
+    const errors = await f("./")
+    let errorsNumber = 0
+    for await (const e of errors) {
+      console.error(e)
+      ++errorsNumber
+    }
+    console.log(`errors: ${errorsNumber}`)
+    return errorsNumber === 0 ? 0 : 1
```

Comment:
```suggestion
    return errorsNumber;
```

or I'm missing something?